writing-tips.md

```diff
@@ -55,3 +55,19 @@ Here are some title examples and their suggested file name structures:
 ## Definitions
 
 If you are repeatedly using terms throughout a large article, you can define them once and then use them repeatedly in the body of the article.
+
+## Writing help for programmers
+These resources will make you better at explaining your products and ideas. 
+
+* [Google Technical Writing Course](https://developers.google.com/tech-writing)
+  > A 1 hour long self-paced course. It is a must-read for every programmer.
```

Comment:
Suggest changing to:

An hour-long...

Also change any other numbers 0 - 9 from numerals to words in this PR in line with the Google style guide.

https://developers.google.com/style/numbers#numbers-as-words